ice/mvc/model.zep

```diff
@@ -402,7 +406,13 @@ abstract class Model extends Arr implements \Serializable
 
         this->di->applyHook("model.before.update", [this]);
 
-        let status = this->db->update(this->from, primary, this->fields(this->getData(), !this->autoincrement));
+        let fields = this->fields(this->getData(), !this->autoincrement);
+
+        if typeof this->primary == "array" {
+            let fields = array_diff_key(fields, primary);
```

Comment:
Maybe we could remove it in the fields()? https://github.com/ice/framework/pull/202/files#diff-adc4d2a2f6f7dca511a6e8073769eb86R266

Reply:
`create` require to store the primary keys